### DISCUSSION TITLE
Turn of PR creation completely

### DISCUSSION THREAD

**Discussion by muellerst-hg** [renovatebot/renovate#19779] [Request Help] [answered]
I would like to turn of PR creation completely, renovate should just create branches.
Our gitea just acts as a bridge and we cannot automate merging.

I could not find a configuration option. Is this possible at all?

Self hosted renovate
Sself hosted platform gitea

**Comment by rarkins** [comment, marked as answer]
I'm not sure we have any trick way to achieve that. Most elegant would be to add a new option prCreation=never (https://docs.renovatebot.com/configuration-options/#prcreation). It's probably not a lot of code to achieve it so feel free to submit a PR
